### DISCUSSION TITLE
Change default SNES core

### DISCUSSION THREAD

**Discussion by jonmcclay** [OnionUI/Onion#493] [Q&A]
When I launch SNES games it defaults to the Beetle Supafaust core. My save file is snes9x 2010. 

I can jump to the retroarch menu and manually open content and choose the core. My safe works perfectly. 

I want to save the snes9x core as default. I have tried saving the configuration in the quick menu and the menu after pressing b. 

I have also tried using the Retroarch app inside the apps folder, then saving the configurations. 

Nothing has changed the default core. Am I missing a step? Should this work?

Here is a video I made trying to capture my steps: https://youtu.be/EHLtUivcpqQ

**Comment by dmolon** [comment]
You can edit the launcher to always open a specific core, just open SNES 'launch.sh' file with notepad and edit this line:

`HOME=/mnt/SDCARD/RetroArch/ $progdir/../../RetroArch/retroarch -v -L $progdir/../../RetroArch/.retroarch/cores/mednafen_supafaust_libretro.so "$1"`

Just change 'mednafen_supafaust_libretro.so' to 'snes9x2010_libretro.so'.
